Module not found: @emoticon/react

When cloning the repository and running the development server for the first time, I'm getting this error:
```
./node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js:4:0
Module not found: Can't resolve '@emotion/react'

Import trace for requested module:
./node_modules/theme-ui/dist/theme-ui.esm.js
./pages/_app.js

https://nextjs.org/docs/messages/module-not-found
```
It seems to be a problem with theme-ui, but I'm not sure.

Steps to reproduce:
```sh
git clone https://github.com/hackclub/theme-starter
cd theme-starter
yarn
yarn dev
```

--- a/package.json
+++ b/package.json
@@ -10,11 +10,14 @@
     "start": "next start"
   },
   "dependencies": {
-    "@hackclub/theme": "^0.3.3",
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
     "@hackclub/meta": "^1.1.32",
+    "@hackclub/theme": "^0.3.3",
     "@mdx-js/loader": "^1.6.22",
-    "next": "^12.0.8",
+    "@mdx-js/react": "^1.6.22",
     "@next/mdx": "^10.0.4",
+    "next": "^12.0.8",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "theme-ui": "^0.13.1"

